### PR TITLE
Readme.md maintenance

### PR DESCRIPTION
**Pull request by clanmills**
[WIP].  Still to rewrite sections 2,3 and 4:

2) Linking your own code to the Exiv2 library
3) Trouble Shooting and Support
4) Building Exiv2 Documentation

### CODE REVIEW

**Comment by piponazo** [comment]
I took the freedom of adding @D4N and myself to the reviewers list ;)

**Comment by piponazo** [comment]
Ups, I just noticed that in this branch you also have the commit that is modifying the README-CONAN.md.  If you want I can drop it and just leave the commit that is touching the README.md

File: README.md [hunk 1]
```diff
@@ -2,177 +2,232 @@
...
 
-On UNIX-like systems, use the GNU configure script. Run the following
-commands from the top directory (containing this file) to configure,
-build and install the library and utility:
 
-    $ ./configure
-    $ make
-    $ sudo make install  (Cygwin/MinGW $ make install)
+<name id="2"></a>
+## 2) Linking your own code with Exiv2
```
Comment (piponazo) [inline README.md]: I think this section has become obsoleted. We are still providing the pkg-config file, but we are also providing now a Config-file with CMake that should ease the consumption of Exiv2 in other projects configured with CMake.

I can take care of updating this section, once this PR gets merged. I have create the issue #359 to not forget about it.

**Comment by piponazo** [comment]
@clanmills I think you made a mistake and pushed the changes requested in #350 in this PR (#352 ). For me it is fine to continue with both changes (for the conan and the main readme files) directly in this PR if that make things easier for you.

**Comment by clanmills** [comment]
I'm going to close this.  I pick up on the changes to the documentation in branch MinGW-0.27Maintenance

**Pull request closed without merging**